apps/src/Sounds.js

```diff
@@ -111,7 +124,7 @@ Sounds.prototype.initializeAudioUnlockState_ = function () {
  */
 Sounds.prototype.isAudioUnlocked = function () {
   // Audio unlock doesn't make sense for the fallback player as used here.
-  return this.audioUnlocked_ || !this.audioContext;
+  return this.audioUnlocked_;
```

Comment:
I don't actually remember.  My best guess is that this case covers the situation described by [this comment](https://github.com/code-dot-org/code-dot-org/blob/63daca8d8c3a00ec6fd1bb826f5e3bb6b69727c0/dashboard/app/assets/javascripts/soundEffects.js#L64), where creating an `AudioContext` fails, but `window.Audio` is still good so we want to assume Audio is unlocked.  I don't have repro steps for that though.  To clarify, that comment is referring to the audio fallback player, not the video fallback player.

If that's the case, you may need to move that try/catch along with the `new AudioContext()` call.

Reading through this file it _looks_ like nothing should work if that `AudioContext` fails to create, but the magic happens in [`Sound.preload`](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/Sounds.js#L556) where it's fine with having no context and will fall back on `window.Audio` if necessary.